Add files via upload (#4395)
influxdata/docs-v2: alert-to-zapier.md

performing certain tasks. In this demonstration, we will be alerting to Discord, one of the most common messaging platforms today.

# Create a Zapier Webhook
Zapier Webhooks are a premium feature, meaning you need to pay for the service. For a payment free 
alternative, it is possible to use the InfluxDB application on the Zapier platform to send alerts for when 
a bucket is listed or data is queried, if that fits your needs. 
1. Log into the Zapier Dashboard and find the “Create Zap” button
2. Name your Zap
3. In the “Trigger” panel, select the Webhook as an event trigger
4. There are three possible events: retrieving a poll, catching a raw hook, or just catching a hook. We will choose the last option in this example.
5. Navigate to the “Set Up Trigger” where you can trigger on specific keys if you desire. In this example, we will leave it blank and continue.
6. In the “Test Trigger” tab, you will be given your webhook, which you will need to save and give to InfluxDB Cloud. You can also test the webhook here as well, but it is not necessary.

{{< img-hd src="static/img/resources/zapier-trigger.png" alt="Create a Zapier Webhook" />}}
Source: Zapier, August 2022

> # Creating an InfluxDB Check
> In this example, we will be using a default threshold check as the task, but it is possible to use your own 
> task written in Flux code. To create a simple check, navigate to the "Alerts" tab on the left and click the "Create" button. Follow the prompts that appear to specify the correct specification for the check. Once the check is completed, navigate to the "Notification Endpoints" tab and create a new endpoint for your alert; if you are a paid InfluxDB user, you will be able to select HTTP as an endpoint instead of the Slack endpoint, or you can configure your endpoint using raw Flux in a similar manner shown the image below.

Reviewer: ```suggestion
## Create an InfluxDB check

[Create an InfluxDB check](https://docs.influxdata.com/influxdb/cloud/monitor-alert/checks/create) to query and alert on a metric you want to monitor.
Use a default **threshold** check as the task. 
_It is possible to use your own task written in Flux code, but for this guide, use the InfluxDB UI to create the check._

Once the check is completed, [create a notification endpoint](https://docs.influxdata.com/influxdb/cloud/monitor-alert/notification-endpoints/create/). Select **HTTP** as an endpoint.
```

> {{< img-hd src="static/img/resources/notification-endpoint.png" alt="Creating a Check" />}}

Reviewer: ```suggestion
{{< img-hd src="static/img/resources/notification-endpoint.png" alt="Create a check" />}}
```

Source: InfluxDB, August 2022

> {{< img-hd src="static/img/resources/flux-endpoint.png" alt="Creating a Check" />}}

Reviewer: Pictures of code aren't really useful. Users will want to copy and paste code.

```suggestion
```

Source: InfluxDB, August 2022

> # Configuring an Action in Zapier
> In the next panel of the Zapier user interface, you can now decide what you want to do when Zapier receives 
> new information from the webhook you just created. You can add multiple actions if you desire, simply by 
> clicking on the plus buttons. In this example, we will only be alerting to Discord.
> 1. Search for the Discord application in the app event panel
> 2. Select the event that you would want to happen when new information reaches the webhook. In this tutorial, 
> I would like to send a channel message
> 3. Fill out the required fields for sending a message 
> 4. From here you will be able to test the messages and publish your Zap

Reviewer: ```suggestion
## Configure a Zapier action

In the Zapier user interface, configure what you want to do when Zapier receives 
new information from the webhook you just created. You can add multiple actions if you desire.
For this example, send an alert to Discord.

1. In the **App Event** panel, search for the **Discord** application.
2. Select the event to trigger when new information reaches the webhook.
   For this example, select **Send Channel Message to Discord**.
3. Provide your Discord credentials. 
4. Test the messages and publish your Zap
```
